math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/tests/FittnessTests.scala

```diff
@@ -47,10 +52,44 @@ object FittnessTests {
 
   // https://en.wikipedia.org/wiki/Mean_squared_error
   def MeanSquareError[R[K] <: RegressorModel[K], K](model: R[K], residuals: DrmLike[K]): R[K] = {
+    // TODO : I think mse denom should be (row - col) ??
     val mse = residuals.assign(SQUARE).sum / residuals.nrow
     model.mse = mse
     model.testResults += ('mse -> mse)
     model.summary += s"\nMean Squared Error: ${mse}"
     model
   }
+
+  // https://en.wikipedia.org/wiki/xxxx
+  def FTest[R[K] <: RegressorModel[K], K](model: R[K],  drmFeatures: DrmLike[K], drmTarget: DrmLike[K]): R[K] = {
+
+    // This is the residual sum of squares for just the intercept
+    //println(" drmTarget.ncol) = " +  drmTarget.ncol)
+    val interceptCol = drmTarget.ncol - 1
```

Comment:
In addition- it seems that this is only used in some print statements that need to be deleted anyway. Probably can drop this line all together